src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Windows.cs

```diff
@@ -26,7 +24,7 @@ public HttpClientHandler()
         {
             if (UseManagedHandler)
             {
-                _managedHandler = new ManagedHandler();
+                _managedHandler = new ManagedHandler() { SslOptions = new SslClientAuthenticationOptions() };
```

Comment:
I find this, and the other one, a bit strange.  Wouldn't that just be something that the ctor already does?

Reply:
I could be convinced to initialize it in the ctor, but I currently have it not doing so due to the thinking that a) someone might not need to set anything on it so why initialize it, and b) someone might want to set it to their own instance so why initialize it.